lib/compare_linker/formatter/base.rb

```diff
@@ -18,10 +25,17 @@ def post_format
       end
 
       def downgrade?(old_ver, new_ver, old_tag, new_tag, old_rev, new_rev)
-        Gem::Version.new(new_ver) < Gem::Version.new(old_ver) ||
+p :bbb
+p new_ver
+p old_ver
+# p Gem::Version.new(new_ver) < Gem::Version.new(old_ver)
+# p (old_tag && new_tag && Gem::Version.new(to_ver(new_tag)) < Gem::Version.new(to_ver(old_tag)))
+# p (old_rev && new_rev && Gem::Version.new(to_ver(new_rev)) < Gem::Version.new(to_ver(old_rev)))
+        (old_ver && new_ver && Gem::Version.new(new_ver) < Gem::Version.new(old_ver)) ||
```

Comment:
I coluld avoid this problem. https://travis-ci.org/masutaka/compare_linker/jobs/429509290

↓ on macOS

```
$ gem --version
2.7.6
```

I don't know the gem version on TravisCI.